lib/Doctrine/ORM/Tools/Console/Command/SchemaTool/UpdateCommand.php

```diff
@@ -109,13 +115,15 @@ protected function configure()
      */
     protected function executeSchemaCommand(InputInterface $input, OutputInterface $output, SchemaTool $schemaTool, array $metadatas)
     {
+        $this->io = new SymfonyStyle($input, $output);
```

Comment:
This could also be part of the `AbstractCommand#execute()`

Reply:
I moved it